.github/workflows/ci.yml

```diff
@@ -88,60 +85,70 @@ jobs:
                 rm -f ~/.gradle/caches/modules-2/gc.properties
 
     instrumentation-tests:
-        runs-on: macOS-latest
-        timeout-minutes: 60
-        env:
-            JAVA_TOOL_OPTIONS: -Xmx4g
-            GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2 -Dkotlin.incremental=false -Dkotlin.compiler.execution.strategy=in-process
-        strategy:
-            fail-fast: false
-            matrix:
-                api-level: [22,25,29]
-        name: Instrumentation tests [API=${{ matrix.api-level }}]
+        name: Instrumentation tests
```

Comment:
Copied from Appyx configuration as the previous RIBs setup was not working (and is very old)